checks/non_committal/new_answer_non_committal.py

```diff
@@ -0,0 +1,82 @@
+from pydantic import ValidationError
+import json
+import openai
+
+from utils.check_schema import LLMToolkitStdCheckInputSchema, LLMToolkitStdCheckOutputSchema, ErrorSchema
+from prompt import detect_noncommittal_response
+from utils.secret_manager import get_secret
+from utils.logger import get_logger
+
+
+logger = get_logger(__name__)
+def lambda_handler(event: dict, context: dict) -> dict:
+    logger.info(f"Going to check if new answer is non committal for {event=}")
+    try:
+        body = LLMToolkitStdCheckInputSchema(**event)
```

Comment:
I apologise for nitpicking here, but maybe we can standardise as referring to the event's input as `input_data` (being used now in the ingestion pipeline) or something similar like `event_input`, which conveys it's identification better than `body`